packages/interpretations/src/components/interpretations/InterpretationDetails.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
Maybe this component isn't needed? Seems all it does is return the Interpretation component.

Reply:
Yes there's no need having this file, i'll just return the Interpretation Component itself instead.